3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -18,28 +18,36 @@ Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-Usin
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
 function requestData(url) {
-  // TODO return a promise using `fetch()`
+  const response = fetch(url);
+  return response
+    .then((data) => data.json())
+    .catch((error) => {
+      console.log(error);
+    });
 }
 
 function renderImage(data) {
-  // TODO render the image to the DOM
   console.log(data);
+  const img = document.createElement('img');
+  img.src = data.img;
+  img.setAttribute('alt', data.alt);
```

Comment:
Like the `.src` property, use can also set the `.alt` property. But `.setAttribute()` works fine too.

```suggestion
  img.alt = data.alt;
```

Reply:
changed as requested